src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -579,12 +579,14 @@ public function formAction($id, Request $request)
             $preview_url_deactive = $adminProductWrapper->getPreviewUrl($product, false);
             $preview_url = $adminProductWrapper->getPreviewUrlDeactivate($preview_url_deactive);
         }
+        
+        $shopIdsList = $shopContext->getContextListShopID();
 
         $attributeGroups = $this
             ->getDoctrine()
             ->getManager()
             ->getRepository('PrestaShopBundle:Attribute')
-            ->findByLangAndShop(1, 1);
+            ->findByLangAndShop($legacyContextService->getContext()->language->id, $shopIdsList[0]);
```

Comment:
Hmm I'm not sure about the `[0]`. It may work for a shop context, but I look at the code of Shop::getShops(), it seems you will never get something at this key:

https://github.com/PrestaShop/PrestaShop/blob/4844543fbcb7652f7769f51639dee37b19dcb237/classes/shop/Shop.php#L754-L757